core/sodasql/cli/cli.py

```diff
@@ -349,6 +350,8 @@ def scan(scan_yml_file: str, warehouse_yml_file: str, variables: tuple = None, t
         scan_builder = ScanBuilder()
         scan_builder.warehouse_yml_file = warehouse_yml_file
         scan_builder.scan_yml_file = scan_yml_file
+        if not TimeHelper.datetime_valid(time):
```

Comment:
Are you sure we need an extra class for this functionality here? Since we catch all the exceptions anyway, this can be just 
```python
datetime.fromisoformat(time)
```

In case of exception it will print exception:
```
soda scan -t blah warehouse.yml tables/demodata.yml
  | 2.1.0b2
  | Scan failed: Invalid isoformat string: 'blah'
Traceback (most recent call last):
  File "/Users/vijay/.pyenv/versions/3.9.4/envs/soda-sql-main/lib/python3.9/site-packages/sodasql/cli/cli.py", line 353, in scan
    datetime.fromisoformat(time)
ValueError: Invalid isoformat string: 'blah'
  | Exiting with code 1
```

Reply:
Oh, also `datetime.isoformat` doesn't support `Z` postfix. 

Either we need to either update the help value that says `The scan time in ISO8601 format like eg 2020-12-31T16:48:30Z`  or  update the code to treat `Z` as `+00:00`

Reply:
@vijaykiran I've just pushed your suggested changes. I went with updating the help text for now, we can probably decide later if we want to treat `Z` as `+00:00`.